Decred Wallet/Features/Transactions/TransactionsViewController.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  TransactionsViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2018-2020 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+import Dcrlibwallet
+
+class TransactionsViewController: UIViewController {
+    @IBOutlet weak var headerTopConstraint: NSLayoutConstraint!
+    @IBOutlet weak var headerStackView: UIStackView!
+    @IBOutlet weak var headerBottomConstraint: NSLayoutConstraint!
+    @IBOutlet var txTableView: UITableView!
+    @IBOutlet var txFilterDropDown: DropMenuButton!
+    @IBOutlet var txSortOrderDropDown: DropMenuButton!
+
+    var noTxsLabel: UILabel {
+        let noTxsLabel = UILabel(frame: self.txTableView.frame)
+        noTxsLabel.text = LocalizedStrings.noTransactions
+        noTxsLabel.font = UIFont(name: "Source Sans Pro", size: 16)
+        noTxsLabel.textColor = UIColor.appColors.lightBluishGray
+        noTxsLabel.textAlignment = .center
+        return noTxsLabel
+    }
+
+    var refreshControl: UIRefreshControl!
```

Comment:
Move this to after `txTableView` declaration and move `txTableView` declaration below the dropdowns declaration to match how the UI displays the dropdowns before the tableview.